base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -1491,6 +1511,243 @@ where
         ))
     }
 
+    #[allow(clippy::too_many_lines)]
+    pub async fn spend_backup_pre_mine_utxo(
```

Comment:
Maybe in another PR (to be prioritized) we can DRY up the code in these functions, as many parts of this function look the same as the one that spends via the m-of-n path.